wemake_python_styleguide/visitors/ast/complexity/offset.py

```diff
@@ -25,16 +25,20 @@
     'visit_Break',
     'visit_Assign',
     'visit_Expr',
+    'visit_Await',
+    'visit_Pass',
+))
+@alias('visit_async_statement', (
     'visit_AsyncFor',
     'visit_AsyncWith',
-    'visit_Await',
+    'visit_AsyncFunctionDef',
```

Comment:
Why this line is required? I am not sure that it actually is.
Because you can not nest functions in other functions. 

That's why I am wondering if we really need `async def` to be checked here.